### DISCUSSION TITLE
Do Quarto websites support hierarchical categories/tags for sidebar navigation?

### DISCUSSION THREAD

**Discussion by the-solipsist** [quarto-dev/quarto-cli#6517] [Q&A] [answered]
### Description

I'm looking to create a website / blog using Quarto to serve as an archive/repository for all my existing writing which are spread across different publications and different formats (newspaper op-eds, reports, journal articles, etc.).  I'd like to create hierarchical categories so that one can select not only on the subject matter categories but also select categories by output type.  I'm imagining something like

`category: [Type/newspaper, Subject/IPR, Subject/copyright]`
translating to a sidebar with:
```
Type
  - newspaper
Subject
  - IPR
  - copyright
```

If there's a different kind of solution for what I'm trying to achieve, please let me know.

**Comment by mcanouil** [comment, marked as answer]
Quarto does not natively supports this, but you can define your own listing page and do nearly whatever you want, see <https://quarto.org/docs/websites/website-listings.html> and <https://quarto.org/docs/websites/website-listings-custom.html>.

**Reply by yairmau** [reply]
hierarchical categories and tags would be wonderful, are there any plans to incorporate those?

**Reply by cscheid** [reply]
No, we are unlikely to do it - the implementation complexity does not justify the scope of the feature.
